src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -130,24 +106,26 @@ As a convenient shorthand, `transition:persist` can alternatively take a transit
 
 ## Built-in Animation Directives
 
-Astro comes with a few built-in animations to override the default `morph` transition. Add the `transition:animate` directive to try Astro's `slide` or `fade` transitions.
+Astro comes with a few built-in animations to override the default `fade` transition. Add the `transition:animate` directive to customize the behavior of your transitions.
 
-- `morph` (default): The browser determines the best way to animate the element depending on how similar the pages are. For example, if the element is positionally different between pages, it will appear to float to its new position. If the element is in the exact same position, it will appear to not move at all.
+- `fade` (default): An opinionated crossfade animation, where the old content fades out and the new content fades in.
+- `initial`: Opt-out of Astro's opinionated crossfade animation and use the browser's default styling.
```

Comment:
I guess since we're this as a verb it should be without the hyphen? Will let @sarah11918 make the final call here!
```suggestion
- `initial`: Opt out of Astro's opinionated crossfade animation and use the browser's default styling.
```

Reply:
Yan knows his verbs!